src/ui/public/agg_table/agg_table.js

```diff
@@ -93,6 +95,35 @@ uiModules
             formattedColumn.class = 'visualize-table-right';
           }
 
+          let isFirstValueNumeric =
+            table.rows && table.rows[0] && table.rows[0][i] && _.isNumber(table.rows[0][i].value);
```

Comment:
Also, you should use (_.get)[https://lodash.com/docs#get] so that way you don't have to write out this long statement.